packages/rocketchat-api/server/api.js

```diff
@@ -159,8 +194,24 @@ class API extends Restivus {
 						});
 
 						logger.debug(`${ this.request.method.toUpperCase() }: ${ this.request.url }`);
+						const requestIp = this.request.headers['x-forwarded-for'];
```

Comment:
@MarcosSpessatto take a look on this https://github.com/RocketChat/Rocket.Chat/blob/a7bc51d51f8f19816e8798e08e2935d205875e8d/packages/rocketchat-cors/cors.js#L107 I think that there are more places where the IP could be